client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -16,7 +16,8 @@ import {
 	getImageEditorTransform,
 	getImageEditorFileInfo,
 	getImageEditorCrop,
-	isImageEditorImageLoaded
+	isImageEditorImageLoaded,
+	getImageMeetsMinimumDimensions
```

Comment:
Oh looking at the code again, we should already have width/height. We can fetch this with `getImageEditorOriginalAspectRatio( state )`.

Also note that we have a special store enhancer set in local dev so we can query for local state/dispatch some actions in console.

<img width="372" alt="screen shot 2017-07-21 at 2 18 15 pm" src="https://user-images.githubusercontent.com/1270189/28482931-93ee81c6-6e1f-11e7-8995-2625ecc67031.png">